### DISCUSSION TITLE
Can not alter column to timestamp from timestamptz

### DISCUSSION THREAD

**Discussion by zulfikra21** [prisma/prisma#17281] [Q&A]
ASK
hello guys, i can not change timestamptz to timestamp without time zone through prisma.
this following my code 

before
model worker_attendance {
attendance_id    Int       @id @default(autoincrement())
  worker_id        Int
  job_id           Int
  check_in         DateTime? @db.Timestamp(6)
  check_out        DateTime? @db.Timestamp(6)
  created_at       DateTime? @default(now()) @db.Timestamptz(6) <---- this code
  updated_at       DateTime? @db.Timestamptz(6)
  country          String?   @default("malaysia") @db.VarChar(40)
  accepted_at      DateTime? @db.Timestamptz(6)
  rejected_at      DateTime? @db.Timestamptz(6)
  check_in_status  String?   @default("pending") @db.VarChar(20)
  check_out_status String?   @db.VarChar(20)
  done             Boolean?  @default(false)
}

to 

model worker_attendance {
attendance_id    Int       @id @default(autoincrement())
  worker_id        Int
  job_id           Int
  check_in         DateTime? @db.Timestamp(6)
  check_out        DateTime? @db.Timestamp(6)
  created_at       DateTime? @default(now()) @db.Timestamp(6) <---- to this code
  updated_at       DateTime? @db.Timestamptz(6)
  country          String?   @default("malaysia") @db.VarChar(40)
  accepted_at      DateTime? @db.Timestamptz(6)
  rejected_at      DateTime? @db.Timestamptz(6)
  check_in_status  String?   @default("pending") @db.VarChar(20)
  check_out_status String?   @db.VarChar(20)
  done             Boolean?  @default(false)
}

**Comment by nurul3101** [comment]
Hello @zulfikra96 👋 
Which database are you using? Are you getting any specific error?

Did you had a chance to look at this native [DateTime mappings](https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#datetime) for your database?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
